.github/workflows/global-api-actions.yml

```diff
@@ -0,0 +1,92 @@
+name: Push Web app to GHCR and deploy to EKS
+
+on:
+  workflow_dispatch:
+
+  push:
+    paths:
+      - global-api/**
+      - k8s/cc-global-api-deploy.yml
+      - k8s/cc-global-api.yml
+    branches: ["develop"]
+
+jobs:
+  runTests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: cd app && npm install
+
+      - name: Set up database
+        run: |
+          docker run --name github_action_postgresql -d -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_PASSWORD="" postgres
+          sleep 10
+          createuser -w -h localhost -p 5432 -U postgres citycatalyst
+          createdb -w -h localhost -p 5432 -U postgres citycatalyst -O citycatalyst
+          cd app
+          cp env.example .env
+          npm run db:migrate
+
+      - name: Run NextJS build
+        run: |
+          cd app
+          npm run build
```

Comment:
This file needs to be adjusted to the global-api project - use python instead of NodeJS, run python code validation (e.g. `black --check` for code style checking, later on tests, build the dockerfile for it at the end